app/pages/_app.tsx

```diff
@@ -1,16 +1,29 @@
 import type { AppProps } from 'next/app';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import localFont from 'next/font/local';
 import '../styles/globals.css';
 import Layout from '@/components/layout/Layout';
 import { CasUserContextProvider } from '@/access/useCasUserContext';
 
+const cinzel = localFont({
+    src: '../styles/fonts/Cinzel-Regular.ttf',
+    variable: '--font-cinzel'
+})
+
+const cormorantUpright = localFont({
+    src: '../styles/fonts/CormorantUpright-Regular.ttf',
+    variable: '--font-cormorant-upright',
+})
```

Comment:
Is it possible to keep the fonts in the globals.css file? I think Next.js's fonts can get messy.